dev/CommonStyles/Deprecated_themeresources.xaml

```diff
@@ -211,6 +225,10 @@
     <Thickness x:Key="ExpanderDropdownDownBorderThickness">1,0,1,1</Thickness>
     <Thickness x:Key="ExpanderDropdownUpBorderThickness">1,1,1,0</Thickness>
     
+    <CornerRadius x:Key="ComboBoxHiglightBorderCornerRadius">8</CornerRadius>
+    <CornerRadius x:Key="ComboBoxDropDownButtonBackgroundCornerRadius">4</CornerRadius>
+    <CornerRadius x:Key="ComboBoxItemCornerRadius">4</CornerRadius>
```

Comment:
This should have a value of 3 if we want to just move the resource into a different file.
```suggestion
    <CornerRadius x:Key="ComboBoxItemCornerRadius">3</CornerRadius>
```

Reply:
All context menus have 4px for items, except for ComboBox for some reason. It was probably an oversight.